cpp/ps0003/main.cpp

```diff
@@ -1,21 +1,23 @@
-
 #include <iostream>
 using namespace std;
 
 int main()
- {
-   int number;
-cout<<"please enter a number "<<endl;
-
-    cin >> number;
-    if (number % 2 == 0)
 {
-        cout << "even" << endl;
-    }
-else
-{
-        cout << "odd" <<endl;
-    }
-
-    return 0;
+   // prompting user for number 2/2
+   int number;
+   cout<<"please enter a number "<<endl;
+   
+   // checking if the number is odd or even 3/3
+   cin >> number;
+   if (number % 2 == 0)
+   {
+      cout << "even" << endl;
+   }
+   else
+   {
+      cout << "odd" <<endl;
+   }

```

Comment:
- [x] used modulo operation by 2 `%2` to check if the number is even or odd
- [x] number is even when modulo result returns `0` and odd when returns `1`
- [x] used `std::cout` in a right way to print result in console